src/Symfony/Component/Serializer/Encoder/XmlEncoder.php

```diff
@@ -81,7 +81,7 @@ public function decode($data, $format, array $context = array())
         libxml_clear_errors();
 
         $dom = new \DOMDocument();
-        $dom->loadXML($data, LIBXML_NONET | LIBXML_NOBLANKS);
+        $dom->loadXML($data, LIBXML_NONET | LIBXML_NOBLANKS | (defined('LIBXML_PARSEHUGE') ? LIBXML_PARSEHUGE : 0));
```

Comment:
This constant always exist because Symfony requires PHP >= 5.3.3. You can remove the test.